### PR TITLE
Add hover effect to social media icons

### PR DESCRIPTION
**Pull request by TBorundia**
# Issue Title: Add hover effect to social media icons
*closes* #457

## Type of change ☑️

What sort of change have you made:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: ☑️
- [X] My code follows the guidelines of this project.
- [X] I have performed a self-review of my own code.
- [X] My changes generate no new warnings.
- [X] I have added things that prove my fix is effective or that my feature works.

## How Has This Been Tested? ⚙️

Describe how have you verified the changes made.

https://github.com/user-attachments/assets/f805e9dc-5fb0-444f-b836-d92717595cf9

### CODE REVIEW

**Comment by TBorundia** [comment]
@singodiyashubham87 Sir,
Please review this PR.
Thank You!

File: package-lock.json
Comment (singodiyashubham87) [inline package-lock.json:1]: You don't have to make changes to this file.

**Review by singodiyashubham87 — CHANGES_REQUESTED** [review]
Review the changes.

**Comment by TBorundia** [comment]
> Review the changes.

@singodiyashubham87 sir, Now I have resolved the conversation .
Please review this.

**Review by singodiyashubham87 — APPROVED** [review]
Approved!

**Pull request merged**